ktcacheorbust.cc

```diff
@@ -38,28 +38,43 @@ void CacheOrBust::configure(kt::TimedDB* dbary, size_t dbnum,
   std::vector<std::string>::iterator it = elems.begin();
   std::vector<std::string>::iterator itend = elems.end();
   while (it != itend) {
-    std::vector<std::string> fields;
-    if (kc::strsplit(*it, '=', &fields) > 1) {
-      const char* key = fields[0].c_str();
-      const char* value = fields[1].c_str();
-      if (!std::strcmp(key, "host")) {
+    std::string param = *it;
+    std::size_t found = param.find("=");
+
+    if (found!=std::string::npos) {
+      std::string key = param.substr(0, found);
+      std::string value = param.substr(found+1);
+
+      if (!key.compare("host")) {
```

Comment:
given that these are all going to be evaluated all the time anyway, might make sense to keep the if-tree for the keys in alphabetic order? easier to scan/maintain like that

Reply:
They are only evaluated once at startup, but sure I can make it alphabetical

Reply:
yeah sorry, that sentence was worded badly. I meant "each of them will pass through this chain", not "continuously looked at"